qiita_pet/handlers/download.py

```diff
@@ -222,3 +230,59 @@ def get(self, study_id):
         self.set_header('Content-Disposition',
                         'attachment; filename=%s' % zip_fn)
         self.finish()
+
+
+class DownloadEBISampleAccessions(BaseHandler):
+    @authenticated
+    @coroutine
+    @execute_as_transaction
+    def get(self, study_id):
+        sid = int(study_id)
+        # Check general access to study
+        study_info = study_get_req(sid, self.current_user.id)
+        if study_info['status'] != 'success':
+            raise HTTPError(405, "%s: %s, %s" % (study_info['message'],
+                                                 self.current_user.email,
+                                                 study_id))
+
+        st = SampleTemplate(sid)
+
+        text = "sample_name\tsample_accession\n%s" % '\n'.join(
+            ["%s\t%s" % (k, v)
+             for k, v in viewitems(st.ebi_sample_accessions)])
+
+        self.set_header('Content-Description', 'text/csv')
+        self.set_header('Expires', '0')
+        self.set_header('Cache-Control', 'no-cache')
+        self.set_header('Content-Disposition', 'attachment; '
+                        'filename=ebi_sample_accessions_study_%s.tsv' % sid)
+        self.write(text)
+        self.finish()
+
+
+class DownloadEBIPrepAccessions(BaseHandler):
+    @authenticated
+    @coroutine
+    @execute_as_transaction
+    def get(self, prep_template_id):
+        pid = int(prep_template_id)
+        pt = PrepTemplate(pid)
+        sid = pt.study_id
+        # Check general access to study
+        study_info = study_get_req(sid, self.current_user.id)
+        if study_info['status'] != 'success':
+            raise HTTPError(405, "%s: %s, %s" % (study_info['message'],
+                                                 self.current_user.email,
+                                                 str(sid)))
+
+        text = "sample_name\texperiment_accession\n%s" % '\n'.join(
+            ["%s\t%s" % (k, v)
+             for k, v in viewitems(pt.ebi_experiment_accessions)])
+
+        self.set_header('Content-Description', 'text/csv')
+        self.set_header('Expires', '0')
+        self.set_header('Cache-Control', 'no-cache')
+        self.set_header('Content-Disposition', 'attachment; '
+                        'filename=ebi_sample_accessions_study_%s.tsv' % sid)
```

Comment:
Can you change the name of the file here so it's not the same as above? I would say something along the lines of `ebi_experiment_accessions_study_%s_prep_%s.tsv`

Reply:
good catch, I forgot to change that!